packages/flutter/lib/src/painting/image_resolution.dart

```diff
@@ -116,7 +116,11 @@ const double _kLowDprLimit = 2.0;
 /// ```dart
 /// AssetImage('icons/heart.png')
 /// ```
+/// See also:
 ///
+///  * [loading images](https://flutter.dev/docs/development/ui/assets-and-images#loading-images-1)
```

Comment:
That link doesn't actually show how to use an AssetImage to get the asset, as far as i can tell. It shows how to use an AssetImage with an Image widget but that's already explained in the "See also" link at the bottom of this section.

What https://github.com/flutter/flutter/issues/78088 is asking for is for sample code showing how to actually get a `ui.Image` object out of an `AssetImage`.